src/main.rs

```diff
@@ -701,6 +730,7 @@ fn backtrace(
     sp_ram_region: &Option<RamRegion>,
     live_functions: &HashSet<&str>,
     current_dir: &Path,
+    force_backtrace: bool
```

Comment:
Clippy already told us that we are using too many arguments 🙈 But this should probably be tackled separately.

Reply:
yeah :D I think there's a lot of rom for refactoring here, tried to make up for this sind by improving some others :)